griptape/drivers/image_query/amazon_bedrock_image_query_driver.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+from typing import TYPE_CHECKING, Any
+from attr import define, field, Factory
+from griptape.artifacts import ImageArtifact, TextArtifact
+from griptape.drivers import BaseMultiModelImageQueryDriver
+from griptape.utils import import_optional_dependency
+import json
+
+if TYPE_CHECKING:
+    import boto3
+
+
+@define
+class AmazonBedrockImageQueryDriver(BaseMultiModelImageQueryDriver):
+    session: boto3.Session = field(default=Factory(lambda: import_optional_dependency("boto3").Session()), kw_only=True)
+    bedrock_client: Any = field(
+        default=Factory(lambda self: self.session.client("bedrock-runtime"), takes_self=True), kw_only=True
+    )
+
+    def try_query(self, query: str, images: list[ImageArtifact]) -> TextArtifact:
+        payload = self.image_query_model_driver.image_query_request_parameters(query, images, self.max_output_tokens)
+
+        response = self.bedrock_client.invoke_model(
+            modelId=self.model, contentType="application/json", accept="application/json", body=json.dumps(payload)
+        )
+
+        response_body = json.loads(response.get("body").read())
+
+        if response_body is None:
+            raise ValueError("Model response is empty")
+
+        try:
+            return self.image_query_model_driver.process_output(response_body)
+        except Exception as e:
+            raise e
```

Comment:
This try/except is redundant if we're just immediately reraising.

Reply:
Ah, I actually meant to update that in the exception case.